src/app/still-image/still-image-playground.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import {
+    ApiResponseData, ApiResponseError, Constants,
+    KnoraApiConnection,
+    LogoutResponse, ReadGeomValue,
+    ReadResource,
+    ReadStillImageFileValue, ReadValue
+} from '@dasch-swiss/dsp-js';
+import { DspApiConnectionToken, Region, StillImageRepresentation } from '@dasch-swiss/dsp-ui';
+import { mergeMap } from 'rxjs/operators';
+
+// TODO: get test data from Knora resource
+class Geom extends ReadValue {
+    geometryString: string;
+}
+
+@Component({
+    selector: 'app-still-image',
+    templateUrl: './still-image-playground.component.html',
+    styleUrls: ['./still-image-playground.component.scss']
+})
+export class StillImagePlaygroundComponent implements OnInit {
+
+    resourceIri = 'http://rdfh.ch/0803/00014b43f902'; // incunabula, TODO: use resource from anything project https://dasch.myjetbrains.com/youtrack/issue/DSP-506
+    loading: boolean;
+
+    stillImageRepresentations: StillImageRepresentation[];
+    caption = 'test image';
+
+    actReg: string;
+
+    constructor(@Inject(DspApiConnectionToken) private knoraApiConnection: KnoraApiConnection) {
+    }
+
+    ngOnInit(): void {
+        this.loading = true;
+
+        this.knoraApiConnection.v2.auth.logout().pipe(
```

Comment:
Why do we call logout here?

Reply:
The image I chose as a test image is publicly accessible wit restricted view permissions. No login needed. I just wanted the page to behave always the same, regardless of what the user did before.